docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -64,6 +64,52 @@ The dashboard is no longer accessible, even with the link, until you make it sha
 
 The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
 
+## Email Sharing
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+
+Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the url.
+
+### Enable email sharing
+
+> **Note:** For Grafana Cloud, you will need to contact support to have the feature enabled.
+
+Add the `publicDashboardsEmailSharing` feature toggle to your `custom.ini` file.
+
+```
+[feature_toggles]
+publicDashboardsEmailSharing = true
+```
+
+### Inviting a viewer
+
+1. Click **Only specified people**.
+2. Enter the email you want to share the public dashboard with.
+3. Click **Invite**.
+4. They will receive an email with a one-time use link. This link must be used within **1 hour** or it will be considered expired. Once the link is used, it will give the viewer access to the public dashboard for **30 days**.
+
+### Viewers requesting access
+
+If a viewer without access tries to navigate to the public dashboard, they will be asked to request access by providing their email. They will receive an email with a new one-time use link if the email they provided has already been invited to view the public dashboard and has not been revoked.
+
+### Revoking access for a viewer
+
+1. Click **revoke** on the viewer you'd like to revoke access for.
+2. This will immediately prevent the user from accessing the public dashboard or using any existing one-time use links they may have.
+
+### Resharing access for a viewer
+
+1. Click **Resend** on the viewer you'd like to reshare the public dashboard with.
+2. This viewer will receive an email with a new one-time use link. This will invalidate all previously issued links for that viewer.
+
+### Access limitations
+
+One-time use links utilize browser cookies, so when a viewer is granted access via a one-time use link, they will only have access through the browser they used to claim the link.
+
+A single viewer cannot generate multiple valid one-time use links. When a user gets a new one-time use link issued, all previous ones are invalidated.
+
+If a Grafana user has read access to the parent dashboard, they can view the public dashboard without needing to be granted access to it.
+
 ## Supported datasources
```

Comment:
```suggestion
## Supported data sources
```

Reply:
With regard to "one-time use link" vs "magic link", whatever we use, it should be consistent in the error messages and in the docs. I'm kinda thinking that in the error messages, we could even drop the word "magic" from those messages and then it would line up with what we have in the docs. I don't feel strongly about which one we use (though I'll check with my team if there's any overall preference), as long as we use the term everywhere the user can see it.

Reply:
@Ijin08 - Can you weigh in on the "one-time use link" vs "magic link" question? Is there a UX preference in terms of how we talk about this in the app/docs?

Reply:
Update: after some discussion, preference from docs team is "one-time use link" so no change to these docs, but I think we should update some of the error message text to match. Instead of "This magic link has expired" just "This link has expired." But will wait for @Ijin08 to add his thoughts.

Reply:
+1 to one-time link, the "magic" thing about magic link could be confusing.

Reply:
We use mostly use `datasources` instead of `data sources`. Should we change all? Not sure how we are using this word in other doc places but in public dashboards at least it's always `datasources`.

Reply:
> We use mostly use `datasources` instead of `data sources`. Should we change all? Not sure how we are using this word in other doc places but in public dashboards at least it's always `datasources`.

Yeah, definitely change it to "data sources". It's two words in the application and in the [docs on the topic](https://grafana.com/docs/grafana/latest/datasources/).

Reply:
> 

RIP all our code

Reply:
> +1 to one-time link, the "magic" thing about magic link could be confusing.

RIP all our code